src/valueObjects/errors/__tests__/BaseError.spec.js

```diff
@@ -0,0 +1,22 @@
+import BaseError from '../BaseError';
+
+describe('valueObjects', () => {

```

Comment:
Nitpick: these outer two describes can be combined since we're inside of both the `valueObjects` and `errors` directories and any tests here would only apply to both.